bin/utest/mmap.c

```diff
@@ -56,53 +53,39 @@ static void mmap_with_hint(void) {
 #undef TESTADDR
 
 static void mmap_bad(void) {
-  void *addr;
   /* Address range spans user and kernel space. */
-  addr = mmap_anon_prw((void *)BAD_ADDR_SPAN, BAD_ADDR_SPAN_LEN);
-  assert_fail(addr, EINVAL);
+  syscall_fail(mmap_anon_prw((void *)BAD_ADDR_SPAN, BAD_ADDR_SPAN_LEN), EINVAL);
   /* Address lies in low memory, that cannot be mapped. */
-  addr = mmap_anon_prw((void *)0x3ff000, 0x1000);
-  assert_fail(addr, EINVAL);
+  syscall_fail(mmap_anon_prw((void *)0x3ff000, 0x1000), EINVAL);
   /* Hint address is not page aligned. */
-  addr = mmap_anon_prw((void *)0x12345678, 0x1000);
-  assert_fail(addr, EINVAL);
+  syscall_fail(mmap_anon_prw((void *)0x12345678, 0x1000), EINVAL);
 }
 
 static void munmap_good(void) {
   void *addr;
-  int result;
 
   /* mmap & munmap one page */
   addr = mmap_anon_prw(NULL, 0x1000);
-  result = munmap(addr, 0x1000);
-  assert_ok(result);
+  syscall_ok(munmap(addr, 0x1000));
 
   /* munmapping again fails */
-  result = munmap(addr, 0x1000);
-  printf("result: %d\n", result);
-  assert(errno == EINVAL);
-  // assert_fail(result, EINVAL);
+  syscall_ok(munmap(addr, 0x1000));
```

Comment:
@franciscozdo Should it return `EINVAL`?

Reply:
No, that was mistake, and I forgot to remove that line when I was cleaning previous changes.